NEWS.txt

```diff
@@ -265,6 +265,9 @@ Deprecation
       The old property is still supported for backward compatibility, but now it is disabled by default.
     - CloudstackSnitch is marked as deprecated and it is not actively maintained anymore. It is scheduled to be removed 
       in the next major version of Cassandra.
+    - Usage of dual ports (native_transport_port together with native_transport_port_ssl) is deprecated and it will be
+      removed in next releases. Please consider to use one port only (native_transport_port) which can support
+      unencrypted as well as encrypted traffic.
```

Comment:
I think we should include some more detail here, especially since this is relevant to client implementors. What do you think of this?

> Usage of dual native ports (native_transport_port and native_transport_port_ssl) is deprecated and will be removed in a future release. A single native port can be used for both encrypted and unencrypted traffic; see CASSANDRA-10559. Cluster hosts running with dual native ports were not correctly identified in the system.peers tables and server-sent EVENTs, causing clients that encrypt traffic to fail to maintain correct connection pools. For more information, see CASSANDRA-19392.